src/components/basicFight.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import Player from './player';
+import './basicFight.css';
 
 const weapons = ['rock', 'paper', 'scissors'];
 
```

Comment:
files should be named after the components they declare, so this file should be Game.jsx ;)

Reply:
change basicFight.css to  Game.css

Reply:
you can do the same for basicFight.jsx -> Game.jsx

Reply:
Done :)